### ISSUE TITLE
Start FX after manual slide change

### ISSUE THREAD

**Issue by mindshaped** [kthornbloom/Smoothslides#19]
Is it possible to always start the FX after a slide change, also when the slide has manually been changed by clicking a prev/next button? Right now the FX just kicks in again once the slide auto-advanced after a manual slide-change. Reason: we'd like to avoid the image standing still completely after a manual slide-change. Thx -N.

**Comment by kthornbloom** [comment]
While I think that sounds like a good idea, there are some problems we'd run into in making that happen.

The effects work by first setting styles on the second to last image. For example, it might scale it up by a factor of 1.2, so that once it appears, it can animate back to "1" for a zoom out effect. 

Because of this, I'd have to basically write the inverse of each effect function to account for the back button, doubling the size of the script. There's also the fact that the slideshow pauses when you mouse over the navigation buttons, so after the effect, it would still have a period where it sits still until mouse-out. 

TL;DR - Possible, but might be more work than it's worth?
